src/common/packager.ts

```diff
@@ -94,7 +97,7 @@ export class Packager {
             if (running) {
                 if (!this.packagerProcess) {
                     Log.logWarning(ErrorHelper.getWarning("Packager is still running. If the packager was started outside VS Code, please quit the packager process using the task manager."));
-                    return Q.resolve<void>(void 0);
+                    return Q.resolve<void>(void 1);
```

Comment:
What's the reason for this change? The expression `void 0` is just a way of producing a "value" of type void; the `void` consumes its argument and doesn't return anything.